scripts/pr-maintenance-cron.sh

```
@@ -0,0 +1,90 @@
+# Crontab entry:
+#   */15 * * * * /mnt/ext-fast/archon/scripts/pr-maintenance-cron.sh >> /tmp/pr-maintenance.log 2>&1
+
+set -euo pipefail
+
+# Cron runs with a minimal PATH (/usr/bin:/bin). archon, gh, bun, git
+# often live in user-local bins; prepend them so the script works from
+# both cron and an interactive shell.
+export PATH="$HOME/.bun/bin:$HOME/.local/bin:/usr/local/bin:$PATH"
+
+# --- Configuration ---
+source /home/asiri/gt/mayor/scripts/lib/archon-projects.sh
```

Shellcheck (0.11.0)

[info] Not following: /home/asiri/gt/mayor/scripts/lib/archon-projects.sh was not specified as input (see shellcheck -x).

(SC1091)